projects/commudle-admin/src/app/feature-modules/sys-admin/components/admin-featured-communities/admin-featured-communities.component.html

```diff
@@ -72,7 +77,7 @@
         <app-community-badge [community]="selectedCommunity"></app-community-badge>
         <button (click)="onSelectionDelete()" ghost nbButton size="small" status="danger">Delete</button>
       </div>
-      <input [(ngModel)]="reason" fullWidth nbInput placeholder="Enter the reason" type="text" />
+      <input [(ngModel)]="reason" fullWidth maxlength="50" nbInput placeholder="Enter the reason" type="text" />
```

Comment:
Let's also add the error-text in case the length goes past 50. It's being used in other components (example username) just for reference.